portfolio/src/main/webapp/index.html

```diff
@@ -124,9 +124,17 @@ <h2>Technical Skills</h2>
 	<section class="sectionRow" id="comments">
 		<h2 class="sectionHeader">Comments</h2>
 		<hr class="round">
-		<!--<div id="commentNumberSelector">
-			<button id= "deleteCommentsButton" class="button" onclick="deleteComments()">Delete All Comments</button>     
-		</div> -->
+		<div id="commentNumberSelector">
+			<label>Language: </label>
+			<select id = "selectedLanguage" onchange="loadComments()">
+				<option value="en">English</option>
+				<option value="fr">French</option>
+				<option value="de">German</option>
+				<option value="es">Spanish</option>
+				<option value="zh">Chinese</option>
+			</select>
+			<!-- <button id= "deleteCommentsButton" class="button" onclick="deleteComments()">Delete All Comments</button> -->
+		</div> 
```

Comment:
For this file, tabs look like they are 4 spaces in Git instead of 2 spaces.
l had the same issue, l just convert tabs to spaces in VSCode and set tab spacing to 2 spaces either before or after this step.

Reply:
Thanks!